lib/FileUploader/FileUploader.test.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import {
+  screen, render, fireEvent,
+} from '@testing-library/react';
+
+import '../../test/jest/__mock__';
+import '../../test/jest/__new_mock__';
+
+import { FileUploader } from './FileUploader';
+
+describe('FileUploader', () => {
+  describe('rendering file uploader with children passed as is', () => {
+    const titleText = 'Title text';
+    const buttonText = 'Button text';
+
+    const onDragEnterMock = jest.fn();
+    const onDragLeaveMock = jest.fn();
+    const onDropMock = jest.fn();
+
+    function flushPromises(ui, container) {
+      return new Promise(resolve => setImmediate(() => {
+        render(ui, { container });
+        resolve(container);
+      }));
+    }
+
+    function dispatchEvt(node, type, data) {
+      const event = new Event(type, { bubbles: true });
+
+      Object.assign(event, data);
+      fireEvent(node, event);
+    }
+
+    function mockData(files) {
+      return {
+        dataTransfer: {
+          files,
+          items: files.map(file => ({
+            kind: 'file',
+            type: file.type,
+            getAsFile: () => file,
+          })),
+          types: ['Files'],
+        },
+      };
+    }
+
+    const renderFileUploader = () => render(
+      <FileUploader
+        title={titleText}
+        uploadButtonText={buttonText}
+        isDropZoneActive={false}
+        onDragEnter={onDragEnterMock}
+        onDragLeave={onDragLeaveMock}
+        onDrop={onDropMock}
+      >
+        <div data-testid="children">Children</div>
+      </FileUploader>
+    );
+
+    const fileUploader = (
```

Comment:
why do you need this? `renderFileUploader` and `render(fileUploader);` works the same way

Reply:
done.